skimage/io/tests/test_io.py

```diff
@@ -8,29 +8,20 @@
 from skimage._shared.testing import assert_array_equal
 
 
-@pytest.fixture
-def urllib_urlopen(monkeypatch):
-    def mockopen(resource_name):
-        class MockOpen:
-            @staticmethod
-            def read():
-                # bytes of a 1x1 jpg
-                return (b'\xff\xd8\xff\xe0\x00\x10JFIF\x00\x01\x01\x00\x00\x01'
-                        b'\x00\x01\x00\x00\xff\xdb\x00C\x00\x03\x02\x02\x02\x02'
-                        b'\x02\x03\x02\x02\x02\x03\x03\x03\x03\x04\x06\x04\x04'
-                        b'\x04\x04\x04\x08\x06\x06\x05\x06\t\x08\n\n\t\x08\t\t'
-                        b'\n\x0c\x0f\x0c\n\x0b\x0e\x0b\t\t\r\x11\r\x0e\x0f\x10'
-                        b'\x10\x11\x10\n\x0c\x12\x13\x12\x10\x13\x0f\x10\x10'
-                        b'\x10\xff\xc0\x00\x0b\x08\x00\x01\x00\x01\x01\x01\x11'
-                        b'\x00\xff\xc4\x00\x14\x00\x01\x00\x00\x00\x00\x00\x00'
-                        b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\t\xff\xc4\x00'
-                        b'\x14\x10\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
-                        b'\x00\x00\x00\x00\x00\x00\xff\xda\x00\x08\x01\x01\x00'
-                        b'\x00?\x00*\x9f\xff\xd9')
-        return MockOpen
-
-    monkeypatch.setattr('urllib.request.urlopen', mockopen)
-
+def one_by_one_jpeg():
```

Comment:
I guess we could also store this as a file or, if it is inlined, as a fixed byte-string.

Reply:
I was having trouble figuring out relative vs absolute paths for the fixture. At least variables/functions I understand.

Reply:
I made it a variable.